base/builder/Dockerfile

```diff
@@ -9,14 +9,16 @@ RUN curl -s -L -O https://github.com/krallin/tini/releases/download/v0.18.0/tini
   && install -m 755 tini /bin/tini \
   && rm tini
 
+# Install newer node the nodejs package includes npm
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+
 # for jupyter lab builds we need npm, there is some weird issue with libssl-dev install that this works around
 RUN apt-get update \
   && DEBIAN_FRONTEND=noninteractive apt-get install -y --fix-missing --no-install-recommends \
-  libssl1.0-dev nodejs-dev \
+  libssl1.0-dev \
   && DEBIAN_FRONTEND=noninteractive apt-get install -y --fix-missing --no-install-recommends \
   libssl-dev \
   && DEBIAN_FRONTEND=noninteractive apt-get install -y --fix-missing --no-install-recommends \
-  npm \
   nodejs \
```

Comment:
in fact this whole block should go, all that libssl-dev was theer because nodejs was failing to install at some instance in time.

Reply:
Ah. We do need to install `nodejs` still, but it installs `npm`.

Reply:
I don't get it

Reply:
Let's install both from curl then, does new npm work with old nodejs?